src/test-routes/merge-patch-json.ts

```diff
@@ -0,0 +1,15 @@
+import { app, json } from "../api";
+
+app.category("vanilla", () => {
+  app.patch("/mergePatchJson/single", "PassMergePatchObject", (req) => {
+    if (req.headers["content-type"] === "application/merge-patch+json" && req.headers["content-length"] === "14") {
```

Comment:
I think it be better to use `containsHeader` expectation so there can also be error message when writting the testserver https://github.com/Azure/autorest.testserver/blob/7cc7cafe7dfc110206222262c72ce6f8425ee124/src/api/request-expectation.ts#L65

Reply:
nice didn't know about it, fixing now. thanks!